### PR TITLE
Add super lint GitHub action

### PR DESCRIPTION
**Pull request by timja**

### CODE REVIEW

**Review by adusumillipraveen — APPROVED** [review]

File: .github/workflows/blank.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+
+      - run: sudo apt-get install cookiecutter
+      - run: cookiecutter ../template-product-infrastructure/ --no-input --config-file ../template-product-infrastructure/cookiecutter-ci.yaml
+
+      - name: Super-Linter
+        uses: github/super-linter@v3.10.0
```
Comment (timja) [inline .github/workflows/blank.yml:29]:
```suggestion
        uses: github/super-linter@v3.10.0
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+          cookiecutter ../template-product-infrastructure/ \
+            --no-input \
+            --config-file \
+            ../template-product-infrastructure/cookiecutter-ci.yaml
+
+      - name: Super-Linter
+        uses: github/super-linter@v3.10.0
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          FILTER_REGEX_INCLUDE: rpetemplate
```
Comment (timja) [inline .github/workflows/ci.yml:36]:
```suggestion
          FILTER_REGEX_INCLUDE: rpetemplate/.*
```

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+      - uses: actions/checkout@v2
+
+      - run: sudo apt-get install cookiecutter
+      - run: |
+          cookiecutter ../template-product-infrastructure/ \
+            --no-input \
+            --config-file \
+            ../template-product-infrastructure/cookiecutter-ci.yaml
+
+      - name: Super-Linter
```
Comment (timja) [inline .github/workflows/ci.yml:32]:
```suggestion
      - name: Super-Linter
```
```suggestion
      - run: ls -lash
      - name: Super-Linter
```

File: .github/workflows/ci.yml [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+            --no-input \
+            --config-file \
+            ../template-product-infrastructure/cookiecutter-ci.yaml
+
+      - run: ls -lash
+      - name: Super-Linter
+        uses: github/super-linter@v3.10.0
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          FILTER_REGEX_INCLUDE: rpetemplate/.*
```
Comment (timja) [inline .github/workflows/ci.yml:37]: this seems not to work or not included in the current version:
https://github.com/github/super-linter/issues/710

https://github.com/github/super-linter/pull/684

Will revisit this PR after the next release

**Pull request closed without merging**